iis-wf/iis-wf-citationmatching/src/main/resources/eu/dnetlib/iis/wf/citationmatching/fuzzy/oozie_app/workflow.xml

```diff
@@ -52,12 +51,36 @@
             <name>sparkExecutorCores</name>
             <description>number of cores used by single executor</description>
         </property>
-        
+        <property>
+            <name>oozieActionShareLibForSpark1</name>
+            <value>$UNDEFINED$</value>
```

Comment:
I've kept this as a parameter, because it's a cluster setting and I've wanted to provide all cluster settings for spark 1 through properties file. I can remove it, but I'd vote to keep it and remove it together with other spark1* params when citationmatching will be moved to spark2.